osidb/migrations/0081_migrate_flawacknowledgment_flawmeta.py

```diff
@@ -0,0 +1,54 @@
+# Written manually on 2023-07-07
+
+# https://docs.djangoproject.com/en/4.2/ref/migration-operations/
+from django.db import migrations
+from ast import literal_eval
+
+def forwards_func(apps, schema_editor):
+    FlawMeta = apps.get_model("osidb", "FlawMeta")
+    FlawAcknowledgment = apps.get_model("osidb", "FlawAcknowledgment")
+    for meta_ack in FlawMeta.objects.filter(type="ACKNOWLEDGMENT"):
+        flaw = meta_ack.flaw
+        name = meta_ack.meta_attr["name"]
+        # Convert None to empty string.
+        affiliation = meta_ack.meta_attr.get("affiliation") or ""
+        # hstore holds bolean values as strings containing True|False
+        # so we need to explicitly convert it to the bolean value
+        from_upstream = literal_eval(meta_ack.meta_attr["from_upstream"])
+
+        # FlawAcknowledgment that already exist are skipped because they
+        # are exactly as up-to-date as FlawMeta ACKNOWLEDGMENT.
+
+        if not FlawAcknowledgment.objects.filter( flaw=flaw, name=name, affiliation=affiliation ).exists():
```

Comment:
Hmm, githooks black apparently ignores migrations. I'll fix it later after review.

Reply:
Fixed.